plugins/wm-actions/wm-actions.cpp

```diff
@@ -267,6 +268,35 @@ class wayfire_wm_actions_t : public wf::plugin_interface_t
         return true;
     };
 
+    wf::activator_callback on_send_to_back = [=] (auto ev) -> bool
+    {
+        return execute_for_selected_view(ev.source, [] (wayfire_view view)
+        {
+            auto ws    = view->get_output()->workspace->get_current_workspace();
+            auto views =
+                view->get_output()->workspace->get_views_on_workspace(ws,
+                    wf::LAYER_WORKSPACE);
+            wayfire_view bottom_view = views[views.size() - 1];
+            if (view != bottom_view)
+            {
+                // Move view to bottom by stacking it below the bottom-most view
+                view->get_output()->workspace->restack_below(view, bottom_view);
+
+                // Change focus to the top-most view
```

Comment:
Wayfire's stacking order isn't necessarily indicative of last-focused order. Instead, the next-to-focus view should be the view with the latest `last_focused_timestamp` (see [here](https://github.com/WayfireWM/wayfire/blob/master/src/output/output.cpp#L89)).

Reply:
Done.